### ISSUE TITLE
Updater: Infinite KNXDisconnectException

### ISSUE THREAD

**Issue by dallmair** [selfbus/software-arm-lib#97]
Happened today for me. Trace is captured, will attach later.

*Console log*
```
Program device at flash address 0x4900 with 256 bytes and CRC32 0xFA4048DA
Sending 256 bytes: 37,6%
......................(92,25 B/s)

Program device at flash address 0x4A00 with 256 bytes and CRC32 0x4424C548
Sending 256 bytes: 39,0%
......................(-0,00 B/s)

Program device at flash address 0x4B00 with 256 bytes and CRC32 0xA4904606
Sending 256 bytes: 40,4%
...................UPDCommand.SEND_DATA 15.15.192 disconnected while awaiting ACK : KNXDisconnectException
UPDCommand.SEND_DATA 15.15.192 disconnected while awaiting ACK : KNXDisconnectException
x
UPDCommand.SEND_DATA 15.15.192 disconnected while awaiting ACK : KNXDisconnectException
UPDCommand.SEND_DATA 15.15.192 disconnected while awaiting ACK : KNXDisconnectException
x
UPDCommand.SEND_DATA 15.15.192 disconnected while awaiting ACK : KNXDisconnectException
UPDCommand.SEND_DATA 15.15.192 disconnected while awaiting ACK : KNXDisconnectException
x
UPDCommand.SEND_DATA 15.15.192 disconnected while awaiting ACK : KNXDisconnectException
UPDCommand.SEND_DATA 15.15.192 disconnected while awaiting ACK : KNXDisconnectException
x
```

**Comment by dallmair** [comment]
Seems to be related to the .hex file as the second attempt failed at the exact same position.

**Comment by Darthyson** [comment]
Happend to me some time ago, sadly without a log. If i remember correct it "recovered" after 10-15min waiting.

**Comment by Darthyson** [comment]
Fixed in 6112483
